### PR TITLE
Update mysql/es external table output row count

### PR DESCRIPTION
**Pull request by Seaven**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Avoid mysql external table always do broadcast join

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 4 / 7 (57.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java|4|7|57.14%|

File: fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java [hunk 1]
```diff
@@ -364,7 +364,8 @@ private Statistics.Builder estimateScanColumns(Table table, Map<ColumnRefOperato
     }
 
     private Void computeNormalExternalTableScanNode(Operator node, ExpressionContext context, Table table,
-                                                    Map<ColumnRefOperator, Column> colRefToColumnMetaMap) {
+                                                    Map<ColumnRefOperator, Column> colRefToColumnMetaMap,
+                                                    int outputRowCount) {
```
Comment (liuyehcf) [inline fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java:368]: new parameter never used?

Comment (Seaven) [reply fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java:368]: Really stupid

**Review by liuyehcf — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java [hunk 2]
```diff
@@ -364,32 +364,37 @@ private Statistics.Builder estimateScanColumns(Table table, Map<ColumnRefOperato
...
 
         context.setStatistics(builder.build());
         return visitOperator(node, context);
     }
 
     @Override
     public Void visitLogicalMysqlScan(LogicalMysqlScanOperator node, ExpressionContext context) {
-        return computeNormalExternalTableScanNode(node, context, node.getTable(), node.getColRefToColumnMetaMap());
+        return computeNormalExternalTableScanNode(node, context, node.getTable(), node.getColRefToColumnMetaMap(),
+                10000);
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java:379]: would better put to StatisticsEstimateCoefficient

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 5 / 8 (62.50%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java|5|8|62.50%|

**Review by Youngwb — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Comment by Seaven** [comment]
@mergeify backport branch-2.2

**Comment by Seaven** [comment]
@mergify  backport branch-2.2

**Comment by Seaven** [comment]
@mergify backport branch-2.2

**Comment by wangruin** [comment]
@mergify backport branch-2.1

**Comment by Seaven** [comment]
@mergify backport branch-2.1

**Pull request merged**